lib/article-tile/index.js

```diff
@@ -2,7 +2,7 @@ import React, { Component, PropTypes } from 'react';
 
 require('./style.css');
 
-export class ArticleTile extends Component {
+export default class ArticleTile extends Component {
```

Comment:
Anyway, my understanding is that `import { thing } from 'modulename';` requires knowing that the module being imported has a property of `thing`, where `import thing from 'modulename';` will take the default export and assign it to the variable `thing`, and would be preferable because it does not rely on knowledge of the internals of the module to know the primary exported object.

So, if and when a module is exporting a single "thing" (or in some cases, may have a main "thing", and some subsidiary properties) then one should export a `default` - in a manner analogous to `module.exports = ...` being the primary export in commonjs modules.